app/views/user_mailer/welcome_email.html.erb

```diff
@@ -36,7 +36,7 @@
       <table class="outer" align="center" style="background-color: #ffffff; border: 1px solid black; border-spacing: 0; color: #333333; font-family: sans-serif; max-width: 600px; width: 100%;">
         <tr>
           <td class="full-width-image center" style="padding: 0; text-align: center;">
-            <%= image_tag attachments['logo.svg'].url, alt: 'home page banner' %>
+              <%= image_tag 'logo.svg', alt: 'home page banner' %>
```

Comment:
Nit: While we're here, can we fix the indentation of the image tag and the `<p>` tag below it to match indentation level used throughout this file please?

```suggestion
            <%= image_tag 'logo.svg', alt: 'home page banner' %>
```